### ISSUE TITLE
`id serial primary key` SQL created by Drizzle in MySQL mode. Dolt can't execute.

### ISSUE THREAD

**Issue by timsehn** [dolthub/dolt#7395]
```sql
create table if not exists `__drizzle_migrations` (
  id serial primary key,
  hash text not null,
  created_at bigint)
```

Errors with:

```
"cannot include more than one key option for a column definition at position 79 near 'key'"
```

I'll try and repro in dolt and mysql now.

**Comment by fulghum** [comment]
`SERIAL` is an alias for `BIGINT UNSIGNED NOT NULL AUTO_INCREMENT UNIQUE`, and we seem to trip over seeing `UNIQUE` and `PRIMARY KEY` together. We could probably just ignore `UNIQUE` when `PRIMARY KEY` is there to unblock this statement.
https://dev.mysql.com/doc/refman/8.0/en/numeric-type-syntax.html

**Comment by jycor** [comment]
related issue:
https://github.com/dolthub/dolt/issues/6903

**Comment by fulghum** [comment]
The fix for this has been released in [Dolt 1.32.2](https://github.com/dolthub/dolt/releases/tag/v1.32.2)
